yt/data_objects/data_containers.py

```diff
@@ -269,10 +268,7 @@ def __getitem__(self, key):
         try:
             rv = self.field_data[f]
         except KeyError:
-            if isinstance(f, tuple):
-                fi = self.ds._get_field_info(*f)
-            elif isinstance(f, bytes):
-                fi = self.ds._get_field_info("unknown", f)
```

Comment:
this is interesting. any idea what the history of using a byte string here is? @matthewturk?